documentation/src/docs/deployment/run-local.md

```diff
@@ -76,9 +76,12 @@ HOST_IP=1.2.3.4 docker-compose up
 ```
 Clients can then connect to the controller at `${HOST_IP}:9090`.
 
-## Configuring standalone
+## Running Pravega in Standalone Mode with SSL/TLS Enabled
+
+By default both the `singlenode.enableTls` and `singlenode.enableauth` are disabled. The configurations, `singlenode.enableTls` and `singlenode.enableauth` can be used to enable encryption and authentication respectively.
```

Comment:
Since this subsection is now about TLS, references to auth related configuration should be avoided here, I believe. What do you say?

Reply:
Also, these sentences can be simplified: "By default, SSL/TLS is disabled. To enable it, set the value of  `singlenode.enableTls` to true. If you enable TLS...., you must also set the following configuration properties: ...."

Reply:
accepted suggestion